config/settings/base.py

```diff
@@ -338,6 +338,9 @@
     "ol",
     "ul",
 ]
+GENERIC_RICH_TEXT_FEATURES = RESTRICTED_RICH_TEXT_FEATURES + [
+    "document-link",
+]
```

Comment:
Will add these settings to some RichText fields/blocks when we have somewhere to use them 🙂

Reply:
Yeah, I think this should probably be more a case-by-case thing as we get more different "types" or rich text areas...